fastmri_recon/helpers/adversarial_training.py

```diff
@@ -117,7 +118,11 @@ def adversarial_training_loop(
         use_multiprocessing=False,
         max_queue_size=10,
         include_d_metrics=False,
+        gen_pre_training_steps=0,
+        pre_training_callbacks=None,
+        # inner_training_gen=1
```

Comment:
I think this is legacy

Reply:
These first two parameters are actually necessary for the pretraining, but I will remove the commented line.